### ISSUE TITLE
Graphics primitives similar to spark.primitives.*

### ISSUE THREAD

**Issue by joshtynjala** [BowlerHatLLC/feathers-sdk#42]
http://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/spark/primitives/package-detail.html

**Comment by devlfm** [comment]
OMG! can't wait for this!  ⭐️ ⭐️ ⭐️ ⭐️ ⭐️

**Comment by devlfm** [comment]
At this time I'm creating LayoutGroups with backgroundSkin="{new Quad(1,1,0x000000)}", and would be great at least the Quad have some type of integration on mxml and layoutData:

```
<f:Rect color="#fefefe">
    <f:layoutData>
        <f:AnchorLayoutData left="0" right="0" top="0" bottom="0"/>
    </f:layoutData>
</f:Rect>
```
I'm also doing lines with Quads... bah..

Worrie: Making backgroundSkin="{}" (bindable) does really bind? or is used internally only once? or should I on initialize() set the backgroundSkin?

**Comment by izzmeifter** [comment]
100% Agree! This would be a welcome addition!

**Comment by teotigraphix** [comment]
Yeah, +1 I do a lot of dynamic stuff and this would help out a lot.

**Comment by teotigraphix** [comment]
@joshtynjala Are you thinking about implementing Path as well? Or just solids?

**Comment by joshtynjala** [comment]
@teotigraphix I haven't put a ton of thought into it. Someone in the forums mentioned that it would be a nice feature, so I added it to the list. I know that starling.display.Canvas has a Polygon class, but certain types of paths don't work super well. I'm sure that if I implemented something like spark.primitives.Path, it would be somewhat limited.

Also, starling.display.Canvas doesn't support strokes yet, so this would only work for fills until Daniel implemented strokes.

**Comment by devlfm** [comment]
I just saw this: https://github.com/gree/tesspathy
